czkawka_core/src/common.rs

```diff
@@ -7,9 +7,9 @@ use std::time::SystemTime;
 pub struct Common();
 impl Common {
     /// Printing time which took between start and stop point and prints also function name
-    pub fn print_time(_start_time: SystemTime, _end_time: SystemTime, _function_name: String) {
+    pub fn print_time(start_time: SystemTime, end_time: SystemTime, function_name: String) {
```

Comment:
I changed names to underscore because clippy or rust in release build talk that this variables are unused, but if this doesn't show now any warnings it will be a little strange.

Reply:
Really, the function should be wrapped in a `#[cfg(debug_assertions)]`, but that would require wrapping all uses of it with `#[cfg(debug_assertions)]`. I would prefer doing that to having underscored variables used, but it is your decision.

Reply:
Putting in each line before `debug_print` a `#[cfg(debug_assertions)]`  is not intuitive, so I think that `#[allow(unused_variables)]` should be added to this function and underscores from variables should be also removed.

After this change CI should pass, and then I will able to check if there isn't any regressions.

Reply:
Fixed in latest commit.